tests/integration/api-gateway/Dockerfile

```diff
@@ -20,15 +20,12 @@ RUN mkdir /test &&\
 	mv ${BASE_DIR}/app.test /test/app.test  &&\
 	mv ${BASE_DIR}/licenses /test/licenses
 
-FROM alpine:3.10
+FROM eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20200310-5f52f407
 LABEL source=git@github.com:kyma-project/kyma.git
 WORKDIR /test
 
 RUN apk update &&\
-	apk add curl bash grep &&\
-	curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl &&\
-	chmod +x kubectl &&\
-	mv kubectl /usr/local/bin/kubectl
+	apk add curl bash grep ca-certificates
```

Comment:
AFAIK, the alpine-kubectl image features curl and bash by default, no need to add them here.